src/amo/css/OverallRating.scss

```diff
@@ -1,39 +1,31 @@
-.OverallRating {
-  form {
+.OverallRating form fieldset {
+  .OverallRating-selectors {
     display: flex;
     justify-content: center;
+  }
 
-    input {
-      appearance: none;
-      opacity: 0;
-      width: 0;
-      height: 0;
-    }
-
-    label {
-      font-size: 0.8em;
-      font-weight: bold;
-      cursor: pointer;
-      display: block;
-      text-align: center;
-      padding-top: 90px;
-      width: 120px;
-      height: 100px;
-      background-size: 85px;
-      background-position: center;
-      background-repeat: no-repeat;
+  button {
```

Comment:
Since the markup is a bit simpler now, it would be nice if this were `.OverallRating-choice` (or `-button` or something) rather than nesting it.